src/test/java/com/happeningnow/controller/ControllerOrganizerTest.java

```diff
@@ -0,0 +1,146 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.repository.OrganizerRepository;
+import com.happeningnow.util.CustomPageImpl;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public class ControllerOrganizerTest {
+
+    @Autowired
+    private TestRestTemplate resTemplate;
+
+    @Autowired
+    private OrganizerRepository organizerRepository;
+
+    private Organizer organizer1;
+
+    private Organizer organizer2;
+
+    @AfterEach
+    public void setUp(){
+        this.organizerRepository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("This controller method should save an organizer")
+    public void save() {
+        UUID id1 = UUID.fromString("a0d3b612-cde9-417d-8c47-b268cc295e80");
+        organizer1 = new Organizer(
+                id1, "Alex Sander", "Developer", "Portugal", Collections.emptyList()
+        );
+
+        HttpEntity<Organizer> saveRequest = new HttpEntity<>(organizer1);
+        ResponseEntity<Organizer> responseEntity = resTemplate.exchange("/organizer/save",
+                HttpMethod.POST,
+                saveRequest,
+                Organizer.class);
+
+        assertThat(responseEntity.getStatusCode().is2xxSuccessful()).isTrue();
+        assertThat(responseEntity.getBody()).isNotNull();
+        Assertions.assertEquals("Alex Sander", responseEntity.getBody().getName());
+        Assertions.assertEquals("Developer", responseEntity.getBody().getDescription());
+        Assertions.assertEquals("Portugal", responseEntity.getBody().getAddress());
```

Comment:
using the repository, assert that the find by id method find an entry with that id in the database